edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/TokenHelper.cs

```diff
@@ -34,5 +38,164 @@ public static bool IsTokenExpired(string hostName, string token)
         }
 
         public static TimeSpan GetTokenExpiryTimeRemaining(string hostName, string token) => GetTokenExpiry(hostName, token) - DateTime.UtcNow;
+
+        private class SharedAccessSignature
```

Comment:
2 points - 
1. Are there any tests for this that we can add? 
2. We should update the code style in this code to match IoT Edge styling.

Reply:
Checked the sdk code (where the class was copied from) does not have tests. However I found "SharedAccessSignatureTest" in Edge, that covers several cases